libs/eslint-plugin-vx/src/rules/gts_array_type_style.ts

```diff
@@ -11,8 +11,6 @@ const rule: TSESLint.RuleModule<
     docs: {
       description:
         'Recommends using short form T[] for simple array types (containing only alphanumeric characters and dots). Recommends using long form Array<T> for complex array types.',
-      recommended: 'stylistic',
-      requiresTypeChecking: false,
```

Comment:
This is just an API change. AFAICT, these properties were just removed and not added elsewhere.